kurtosis-module/impl/module.go

```diff
@@ -90,8 +94,34 @@ var mergeDevnet3BootnodeEnodes = []string{
 	"enode://46b2ecd18c24463413b7328e9a59c72d955874ad5ddb9cd9659d322bedd2758a6cefb8378e2309a028bd3cdf2beca0b18c3457f03e772f35d0cd06c37ce75eee@137.184.213.208:30303",
 }
  */
+// Defines the strings users can use to define the types of EL clients the participant network will contain
+var elClientKeywords = map[string]participant_network.ParticipantELClientType{
+	gethClientKeyword: participant_network.ParticipantELClientType_Geth,
+	nethermindClientKeyword: participant_network.ParticipantELClientType_Nethermind,
+}
+// Defines the strings users can use to define the types of CL clients the participant network will contain
+var clClientKeywords = map[string]participant_network.ParticipantCLClientType{
+	nimbusClientKeyword: participant_network.ParticipantCLClientType_Nimbus,
+	"lighthouse": participant_network.ParticipantCLClientType_Lighthouse,
```

Comment:
These keys are string instead of constants such as "nimbusClientKeyword"

Reply:
Fixed!